src/components/Header.js

```diff
@@ -0,0 +1,95 @@
+import React, { Component } from 'react'
+import {
+  Collapse,
+  NavbarBrand,
+  NavbarToggler,
+  NavItem,
+  NavLink,
+  Nav,
+  DropdownToggle,
+  DropdownMenu,
+  DropdownItem,
+  UncontrolledDropdown,
+  NavbarText,
+  Navbar
+} from 'reactstrap'
+
+export class Header extends Component {
+  constructor(props) {
+    super(props);
+    this.toggle = this.toggle.bind(this);
+    this.state = { collapse: false };
+  }
+
+  toggle() {
+    this.setState({ collapse: !this.state.collapse });
+  }
+  render() {
+    return (
+      <div>
+        <Navbar
+          color="dark"
+          container
+          dark
+          expand="md"
+          fixed="top"
+        >
+          <NavbarBrand href="/">
+            barkbuds 🐾
+          </NavbarBrand>
+          <NavbarToggler onClick={this.toggle} />
+          <Collapse isOpen={this.state.collapse} navbar>
+            <Nav
+              className="me-auto"
+              navbar
+            >
+              <NavItem>
+                <NavLink href="/">
+                  Home
+                </NavLink>
+              </NavItem>
+              <NavItem>
+                <NavLink href="/dogindex">
+                  All Dogs
+                </NavLink>
+              </NavItem>
+              <NavItem>
+                <NavLink href="/dognew">
+                  New Dog
+                </NavLink>
+              </NavItem>
+              {/* <UncontrolledDropdown
+                inNavbar
+                nav
+              >
+                <DropdownToggle
+                  caret
+                  nav
+                >
+                  Options
+                </DropdownToggle>
+                <DropdownMenu end>
+                  <DropdownItem>
+                    Option 1
+                  </DropdownItem>
+                  <DropdownItem>
+                    Option 2
+                  </DropdownItem>
+                  <DropdownItem divider />
+                  <DropdownItem>
+                    Reset
+                  </DropdownItem>
+                </DropdownMenu>
+              </UncontrolledDropdown> */}
```

Comment:
If you are not using this material please delete and delete the imports for it that are unused.

Reply:
We left it for now in case we wanted a dropdown option in our navigation bar. Will definitely delete when we know for sure we don't want to use it. We made a todo to come back to it. Thanks for the heads up!